doc/website/concepts/qos-policies.md

```diff
@@ -0,0 +1,66 @@
+# QoS policies
+
+Unlike other middlewares, e.g. [DDS used in ROS 2](https://docs.ros.org/en/galactic/Concepts/About-Quality-of-Service-Settings.html),
+iceoryx keeps things simple and offers basic Quality of Service (QoS) settings. The QoS
+settings are called options and can be used to optimize and tailor the communication.
+
+## `ServerOptions` and `ClientOptions`
+
+The two most important settings are:
+
+| Option                                 | Explanation                                                                  |
+|----------------------------------------|------------------------------------------------------------------------------|
+| `ServerOptions::requestQueueCapacity`  | This enables servers to store n requests before they are passed to the users |
+| `ClientOptions::responseQueueCapacity` | This enables clients to store n respones before they are passed to the users |
+
+## Server and client matching criteria
+
+Two criteria have to be fulfilled in order for a server and a client to be connected.
+
+1. Same `capro::ServiceDescription`
+2. Matching `ConsumerTooSlowPolicy` and `QueueFullPolicy` in `ServerOptions`/`ClientOptions`
+
+### Compatible policies
```

Comment:
Just a food for thought. Only this section talks about compatibility, everywhere else matching is used. Would it make sense to call this `Policy matching`?

Maybe also adding a sentence like
```
The `ConsumerTooSlowPolicy` and `QueueFullPolicy` are used to control the behavior for full queues. Since some combinations would be contradicting, not all of them match to connect producer and consumer.
```

Reply:
Added the sentece, but left the title as it is. "Compatible policies" is easier to understand for me.